### PR TITLE
EIP-3198 Draft -> Review

### PR DESCRIPTION
**Pull request by abdelhamidbakhta**
As discussed during ACD we change EIP-3198 Status to Review

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 3198 state from Draft to Review

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Needs an Abstract.  Consider renaming Simple Summary to Abstract and then writing a new Simple Summary like "Adds an opcode that gives the EVM access to the block's base fee.".

----

Needs Rationale to be filled out.  I recommend at least discussing why you decided on the chosen gas cost.

----

Needs Security Considerations.  If there are no known security considerations mention that.

----

Recommend rewording Backward Compatibility to "There are no known backward compatibility issues with this opcode." to match phrasing in other EIPs.

**Comment by abdelhamidbakhta** [comment]
> Needs an Abstract. Consider renaming Simple Summary to Abstract and then writing a new Simple Summary like "Adds an opcode that gives the EVM access to the block's base fee.".
> 
> Needs Rationale to be filled out. I recommend at least discussing why you decided on the chosen gas cost.
> 
> Needs Security Considerations. If there are no known security considerations mention that.
> 
> Recommend rewording Backward Compatibility to "There are no known backward compatibility issues with this opcode." to match phrasing in other EIPs.

Thanks for your review. I addressed your comments. Let me know if there is anything to change.

File: EIPS/eip-3198.md [hunk 1]
```diff
@@ -31,11 +32,16 @@ Add a `BASEFEE` opcode at `(0x48)`, with gas cost `G_base`.
...
+The value of the base fee is needed to process transactions. That means it's value is already available before running the EVM code.
+The opcode does not add extra complexity and additional read/write operations, hence the choice of `G_base` gas cost.
+
 ## Backwards Compatibility
-This EIP is backwards-compatible.
+There are no known backward compatibility issues with this opcode.
 
 ## Test Cases
 
+
```
Comment (lightclient) [inline EIPS/eip-3198.md:44]: Probably don't need this extra space.

**Review by lightclient — APPROVED** [review]

File: EIPS/eip-3198.md (hunk 1 above)
Comment (MicahZoltu) [inline EIPS/eip-3198.md:44]:
```suggestion
```

**Pull request merged**